### ISSUE TITLE
ETI mining failed

### ISSUE THREAD

**Issue by Walk3n** [TrailingStop/TT-Miner-release#31]
Hi TS team, 

I have this error with the command line : 
TT-Miner -c ETI -P ETI 0xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.xxxxxx@eticapool.com:8081

![github](https://github.com/TrailingStop/TT-Miner-release/assets/152641681/9ced91a1-94ac-4a21-a2b1-13dff570bbab)

Can you help me?
Thank you

**Comment by TrailingStop** [comment]
Hi, can you please giver this beta version a try and let me know if it works?

www.TradeProject.de/download/Miner/TT-Miner-2023.4.4b1.zip

**Comment by Teleloco** [comment]
This beta doesnt have linux binary. I would love to use TT-miner in my rig's but the last version doest work in hiveOS.

**Comment by TrailingStop** [comment]
Hi- was under impression from your screenshot that it is windows. Will make a new beta later today or tomorrow.

**Comment by Teleloco** [comment]
No, I'm completely another person. Just doesnt wanted to create another ticket.

**Comment by TrailingStop** [comment]
ah - ok. got it.

**Comment by vasiachumak1** [comment]
www.TradeProject.de/download/Miner/TT-Miner-2023.4.4b1.zip

this version works unstable, the first time everything is fine, and then everything - for 2-3 hours there may be no proof

**Comment by TrailingStop** [comment]
do you have a screenshot or better the log?

**Comment by vasiachumak1** [comment]
![996633](https://github.com/TrailingStop/TT-Miner-release/assets/129108079/ecb9f992-90a8-420f-9612-341eaf2113f2)
this version works unstable, the first time everything is fine, and then everything - for 2-3 hours there may be no proof
[Uploading TT-Pool(eticapool.com-8081)-[20231201].log…]()

**Comment by TrailingStop** [comment]
What do you mean with "unstable"? Is this something I can see? Looks OK for me.

**Comment by TrailingStop** [comment]
Hi - what does not work? Maybe I can help. Can you provide more information? Keep in mind to remove the http:// from the pool url!

**Comment by Teleloco** [comment]
> Hi - what does not work? Maybe I can help. Can you provide more information? Keep in mind to remove the http:// from the pool url!

Where i can find version X.4.4?

**Comment by TrailingStop** [comment]
4.4 is still in beta - not release. working on the release right now. I will release it here on github.

**Comment by Teleloco** [comment]
Oh, ty. Will wait to try. Hope there maybe an improvement in hashrate. Now using 2023.4.3, dual mining ETI+ZIL and works like a charm.

**Comment by killerdrunk** [comment]
> Привет команде TS,
> 
> У меня такая ошибка в командной строке: TT-Miner -c ETI -P ETI [0xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.xxxxxx@eticapool.com](mailto:0xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.xxxxxx@eticapool.com): 8081
> 
> ![github](https://private-user-images.githubusercontent.com/152641681/287337054-9ced91a1-94ac-4a21-a2b1-13dff570bbab.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE2ODM4ODIsIm5iZiI6MTcwMTY4MzU4MiwicGF0aCI6Ii8xNTI2NDE2ODEvMjg3MzM3MDU0LTljZWQ5MWExLTk0YWMtNGEyMS1hMmIxLTEzZGZmNTcwYmJhYi5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwNFQwOTUzMDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZWU5YmUyODc1ZTRmNzllMmFkMzk3MWJhNGRkYWM0YTI2OTUxZmQyNmUyYzE5NmVhNjk4MmI0NmFjZDA2YTVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jWm_qMLpSoVVyacj9JlDvvAGndi4GpDkotCrnE6whp8)
> 
> Вы можете мне помочь? Спасибо

-cETI ETI -PETI без пробелов!
